packages/pwa-kit-dev/bin/pwa-kit-dev.js

```diff
@@ -202,7 +202,17 @@ const main = async () => {
         )
         .addOption(new program.Option('--noHMR', 'disable the client-side hot module replacement'))
         .action(async ({inspect, noHMR}) => {
-            execSync(`node${inspect ? ' --inspect' : ''} ${resolvedSSRPath}`, {
+
+            // Q: Why is all pwa-kit-dev commands using path.join('..', '..', '..')?
```

Comment:
Is it because the upward traversal is meant to find the files it's looking for in alternate locations for cases like CI?

Reply:
Let's not leave this comment in our codebase though, please remove before merging. If we want to make a change, that's fine but let's create a ticket for it if that's the case